texar/torch/core/layers.py

```diff
@@ -515,47 +512,27 @@ class instance, or a `dict` of hyperparameters of respective
     return layer
 
 
-class _ReducePool1d(nn.Module):
```

Comment:
Why was this class removed?

Reply:
My guess is that the logic of `MaxReducePool1d` and `AvgReducePool1d` are pretty straightforward. We do not need to define this additional class `_ReducePool1d`.

Reply:
Pengzhi's right. Also, the previous implementation was hard to type-annotate under the new version.